src/Umbraco.Tests/PropertyEditors/MultiValuePropertyEditorTests.cs

```diff
@@ -123,5 +123,18 @@ public void DropDownPreValueEditor_Format_Data_For_Editor()
             var json = JsonConvert.SerializeObject(result);
             Assert.AreEqual("{\"items\":{\"1\":{\"value\":\"Item 1\",\"sortOrder\":1},\"2\":{\"value\":\"Item 2\",\"sortOrder\":2},\"3\":{\"value\":\"Item 3\",\"sortOrder\":3}}}", json);
         }
+
+        [Test]
+        public void FromConfigurationEditor_WithSerializedConfiguration_DoesReturnCorrectValueListConfiguration()
+        {
+            var serializedJsonInput = "{\"items\":{\"1\":{\"value\":\"Item 1\",\"sortOrder\":1},\"2\":{\"value\":\"Item 2\",\"sortOrder\":2},\"3\":{\"value\":\"Item 3\",\"sortOrder\":3}}}";
+            var inputDictionary = JsonConvert.DeserializeObject(serializedJsonInput, typeof(IDictionary<string, object>));
```

Comment:
why don't use use the generic `DeserializeObject<Dictionary<string, object>>`?

Reply:
Good catch 👍